src/app/static/src/app/store/translations/locales.ts

```diff
@@ -891,7 +900,7 @@ const pt: Partial<Translations> = {
     projectsHeaderCreate: "Criar um novo projeto",
     projectsHeaderReturn: "regressar ao projeto atual",
     projectsNoSelfShare: "Os projetos não podem ser partilhados com a conta do próprio utilizador",
-    promoteVersionHeader: "A copiar versão {{ versão }} para um novo projeto",
+    promoteVersionHeader: "A copiar versão {{ version }} para um novo projeto",
```

Comment:
The {{}} here is for interpolation so shouldn't be translated

Reply:
oops!